app/src/main/java/ahhhlvin/c4q/nyc/vinecodingproject/IssuesActivity.java

```diff
@@ -1,158 +0,0 @@
-                URL endpointUrl = new URL(url);
-                conn = (HttpURLConnection) endpointUrl.openConnection();
-                conn.setRequestMethod(REQUEST_METHOD_GET);
-                conn.setDoInput(true);
-
-                conn.connect();
-                int response = conn.getResponseCode();
-                Log.d("DEBUG_TAG", "The response is: " + response);
-                is = conn.getInputStream();
-                mResult = InputStreamToString(is);
-                is.close();
-            } finally {
-                if (conn != null) {
-                    conn.disconnect();
-                }
-
-                if (is != null) {
-                    is.close();
-                }
-            }
-            return mResult;
-        }
-
-        private String InputStreamToString(InputStream is) throws IOException {
-            BufferedReader br = new BufferedReader(new InputStreamReader(is));
-            StringBuilder sb = new StringBuilder();
-            String line;
-            while ((line = br.readLine()) != null) {
-                sb.append(line);
-            }
-            br.close();
-            return sb.toString();
-        }
-
-
-
-
-        @Override
-        protected ArrayList<GitIssue> doInBackground(Void... arg0) {
-
-            try {
-                JSONArray issueArray = new JSONArray(run(ENDPOINT));
-
-                    for (int i = 0; i < issueArray.length(); i++) {
-                        JSONObject issueObj = issueArray.getJSONObject(i);
-                        GitIssue issue = new GitIssue();
-                        issue.setmIssueTitle(issueObj.get("title").toString());
-                        issue.setmIssueBody(issueObj.get("mBody").toString() + "\n\n");
-
-                        JSONArray commentsArray = new JSONArray(run(issueObj.get("comments_url").toString()));
-                        if (commentsArray.length() > 0) {
-                            for (int j = 0; j < commentsArray.length(); j++) {
-                                JSONObject commentObj = commentsArray.getJSONObject(j);
-                                IssueComment comment = new IssueComment();
-                                comment.setmBody("Posted by: " + commentObj.getJSONObject("user").getString("login").toUpperCase() + "\n\n" + commentObj.getString("mBody") + "\n\n\n");
-                                issue.mCommentsList.add(comment);
-                            }
-                        }
-                        mIssuesList.add(issue);
-                    }
-
-            } catch (JSONException e) {
-                e.printStackTrace();
-            } catch (IOException e) {
-                Log.e("ServiceHandler", "Error retrieving data from URL");
-            }
-            return mIssuesList;
-        }
-
-
-        @Override
-        protected void onPostExecute(ArrayList<GitIssue> list) {
-            super.onPostExecute(list);
-            mAdapter.notifyDataSetChanged();
```

Comment:
then update the new data like  mAdapter.addIssues(issues);
then inside of addIssues method (in adapter), call notifyDataSetChanged();